### ISSUE TITLE
enable access to validation_level=min on data imports

### ISSUE THREAD

**Issue by gregcaporaso** [qiime2/qiime2#721]
To speed up import of very large data sets (e.g., collections of `.fastq.gz` files), we should enable users to change the format validation level on import. This functionality already exists on the formats, but isn't accessible through the existing import functionality.

**Comment by colinvwood** [comment]
I believe this was closed by #734

**Comment by gregcaporaso** [comment]
Confirmed, thanks @colinvwood. 

```shell
qiime tools import --help
Usage: qiime tools import [OPTIONS]

  Import data to create a new QIIME 2 Artifact. See https://docs.qiime2.org/
  for usage examples and details on the file types and associated semantic
  types that can be imported.

Options:
  --type TEXT             The semantic type of the artifact that will be
                          created upon importing. Use `qiime tools list-types`
                          to see what importable semantic types are available
                          in the current deployment.                [required]
  --input-path PATH       Path to file or directory that should be imported.
                                                                    [required]
  --output-path ARTIFACT  Path where output artifact should be written.
                                                                    [required]
  --input-format TEXT     The format of the data to be imported. If not
                          provided, data must be in the format expected by the
                          semantic type provided via --type. Use `qiime tools
                          list-formats --importable` to see which formats of
                          input data are importable.
  --validate-level [min|max]
                          How much to validate the imported data before
                          creating the artifact. A value of "max" will
                          generally read the entire file or directory, whereas
                          "min" will not usually do so.       [default: "max"]
  --help                  Show this message and exit.
```
